base/setup/usetup/format.c

```diff
@@ -96,6 +96,8 @@ FormatPartition(
     if (!FileSystem || !FileSystem->FormatFunc)
         return STATUS_NOT_SUPPORTED;
 
+    CONSOLE_PrintTextXY(7, 11, "100%c", ' ');
+
```

Comment:
`CONSOLE_PrintTextXY()` will not clear the specified text from the screen, that's for sure. The function you might be looking for is `CONSOLE_ClearTextXY()`.

Reply:
https://jira.reactos.org/browse/CORE-12683?focusedCommentId=104847&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-104847

Reply:
That's funny... The function code of `CONSOLE_ClearTextXY()` is implemented as it can be seen from `consup.c` (from 285 to 302 line) but there isn't a prototype declaration of the said function in `consup.h` file header. 

The thing is, making a prototype of this function would create another compile error as the function is static and defining prototypes as static is considered a bad practice, from what I have heard. Anyone knows for which reason that function was declared as static?

Reply:
The function is declared `static` because it is only used in this one .c file. By declaring it `static`, it is not available outside this file.

Reply:
So @eersoy93 can remove the static attribute and add the function prototype in the consup.h.

Reply:
@HBelusca I'm going to do this.